cves/kernel/CVE-2014-8173.yml

```diff
@@ -55,7 +55,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A NULL pointer dereference flaw was caught in the way the
+  Linux kernels madvise MADV_WILLNEED function handled the page table
+  locking. A local, unpriviledged user could use this flaw to crash
+  the system.
```

Comment:
This description is a bit confusing. I'm not sure what the first sentence is saying.